package.json

```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "setup": "yarn && cd front-end/ && yarn && yarn build && cd .. && node app.js"
+    "setup": "yarn && cd front-end/ && yarn && yarn build && cd .. && node app.js",
+    "dockerize": "docker build -t oskardamkjaer/matador . && docker run -it -v ${PWD}:/usr/src/app -v /usr/src/app/node_modules -p 3000:3000 --rm oskardamkjaer/matador"
```

Comment:
probably don't want to name it `oskardamkjaer`/... if we plan on others using it :P Would also recommend you to look into docker-compose, makes managing docker-containers more managable (i.e. create a `docker-compose.yaml` file, instead of this command). YMMV.

Reply:
Also, might be nice to add this to the README :)